roles/ceph-osd/tasks/bcache.yml

```diff
@@ -1,4 +1,35 @@
 ---
+# Create hybrid bucket
+- name: Create root hybrid-bucket root
+  shell: ceph osd crush add-bucket {{ hybrid_pool }} root
+  run_once: true
+
+# Create crush ruleset for the hybrid root bucket
+- name: Create crush ruleset hybrid root bucket
+  shell: ceph osd crush rule create-simple {{ hybrid_ruleset }} {{ hybrid_pool }} host firstn
+  run_once: true
+
+# Create osd host bucket
+- name: Create osd host bucket
+  shell: ceph osd crush add-bucket {{ ansible_hostname }} host
+
+# Move the host bucket under corresponding root
+- name: Move host bucket under root bucket
+  shell: ceph osd crush move {{ ansible_hostname }} root={{ hybrid_pool }}
+
```

Comment:
What happens when these tasks are ran a second time? Seems like there would be failures, or excessive actions.